macros/anaUltraThreshold.py

```diff
@@ -7,7 +7,9 @@
 from PanChannelMaps import *
 from gempython.utils.nesteddict import nesteddict as ndict
 
-parser = OptionParser()
+from qcoptions import parser
+
+#parser = OptionParser()
```

Comment:
Needed for option --vfatmask.  Can move specifically into this file however since it's already present in qcoptions makes sense to have it here.

If scan is taken with a vfatmask!=0 then trying to analyze that data with vfatmask == 0 causes seg fault.

Please advise.